unstructured/ingest/main.py

```diff
@@ -63,6 +65,8 @@ def cleanup(self):
     def _filter_docs_with_outputs(self, docs):
         num_docs_all = len(docs)
         docs = [doc for doc in docs if not doc.has_output()]
+        if self.max_docs is not None:
+            docs = docs[: min(len(docs), self.max_docs)]
```

Comment:
```suggestion
            docs = docs[:self.max_docs]
```
this works. also, for this to not be a stack trace, you need to add `type=int`

Reply:
i think there should be another line here:
```
            num_docs_all = self.max_docs if num_docs_all > self.max_docs
```
to avoid misleading output like:
```
2023-03-28 17:07:43,057 MainProcess INFO     Skipping processing for 1 docs out of 3 since their structured outputs already exist
```

when no prior structured outputs exist.